changelogs/fragments/58260-na_ontap_gather_facts-lun_info.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
```

Comment:
So this isn't needed for this PR. A second PR is needed to merge this in to 2.8.X where this would be needed. 
https://docs.ansible.com/ansible/latest/community/development_process.html#backporting-merged-prs

Reply:
Removed changelog fragment